site/en/r1/tutorials/sequences/text_generation.ipynb

```diff
@@ -98,7 +98,7 @@
         "To watch the next way with his father with his face?\n",
         "\n",
         "ESCALUS:\n",
-        "The cause why then we are all resolved more sons.\n",
+        "The cause why then us all resolved more sons.\n",
```

Comment:
@abhigyadufare Please check if this is part of the sample output of the model, and not something that should be formatted because of the docs style guide.

Reply:
This is NBD, and will ensure that the checks pass (they are just warnings not hard blockers here).